ci-operator/step-registry/openshift/e2e/openstack/byon/openshift-e2e-openstack-byon-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: openshift-e2e-openstack-byon
+  steps:
+    pre:
+      - chain: ipi-openstack-pre
+    test:
+      # The current test suites don't work fine when going through a proxy, so for now
+      # we just want to deploy a cluster and consider it tested. This will be improved 
+      # in the future.
+      - ref: openstack-noop
```

Comment:
I'm pretty sure `openshift-e2e-test` runs well with proxy, at least minimal setup does.

Reply:
would you mind if I try as a follow up patch? I'll test it right after this patch has landed. Thanks!

Reply:
Sure, that sounds good

Reply:
@vrutkovs https://github.com/openshift/release/pull/20577